src/meta_schedule/search_strategy/evolutionary_search.cc

```diff
@@ -719,9 +719,36 @@ class EvolutionarySearch : public SearchStrategy {
                                                     EvolutionarySearchNode);
 };
 
+Array<Schedule> EvolutionarySearchSampleInitPopulation(ObjectRef _self, int num) {
+  const EvolutionarySearchNode* self = _self.as<EvolutionarySearchNode>();
```

Comment:
I added the `EvolutionarySearch` class in the latest PR so you may use it directly now

```suggestion
Array<Schedule> EvolutionarySearchSampleInitPopulation(EvolutionarySearch self, int num) {
```

Reply:
Aha thanks! It's much more convenient.